### PR TITLE
Add pagination for tx history

### LINKED ISSUE #352: Wallet - asset detail: Add pagination for tx history
Currently we do load all data of tx history for last 90 days at once. Change it to load data by using pagination. Hint: `asgardex-binance` provides [`GetTxsParams` to set `limit` and `offset` parameters](https://gitlab.com/thorchain/asgardex-common/asgardex-binance/-/blob/master/src/types/binance.ts#L370-380).

### PR DESCRIPTION
**Pull request by veado**
- [x] Update Binance services to provide pagination
- [x] Add `pagination` to `TransationTable`
- [x] Style `Pagination` component

Closes #352

### CODE REVIEW

File: src/renderer/components/wallet/AssetDetails.tsx [hunk 1]
```diff
@@ -139,6 +141,14 @@ const AssetDetails: React.FC<Props> = (props: Props): JSX.Element => {
     )
   }, [menuItems])
 
+  const onChangePagination = useCallback(
+    (pageNo) => {
+      loadSelectedAssetTxsHandler({ limit: MAX_PAGES, offset: (pageNo - 1) * MAX_PAGES })
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/wallet/AssetDetails.tsx:146]: im really can not understand why offset equals `(pageNo -  1) * MAX_PAGES`. could you pls clarify ? 
seems `MAX_PAGES` is a constant which tells how much elements we allowed to show per page ?

File: src/renderer/services/binance/service.ts [hunk 2]
```diff
@@ -263,30 +264,45 @@ const { get$: selectedAsset$, set: setSelectedAsset } = observableState<O.Option
...
+  // 90 day window - similar to ASGARDEX wallet approach,
   // see https://gitlab.com/thorchain/asgard-wallet/-/blob/develop/imports/api/wallet.js#L39-48
-  const offset = 90 * 24 * 60 * 60 * 1000
-  const startTime = endTime - offset
-  return Rx.from(client.getTransactions({ txAsset, endTime, startTime })).pipe(
-    mergeMap(({ tx }: TxPage) => Rx.of(RD.success(tx))),
+  const diffTime = 90 * 24 * 60 * 60 * 1000
+  const startTime = endTime - diffTime
+  return Rx.from(client.getTransactions({ txAsset, endTime, startTime, limit, offset })).pipe(
+    mergeMap((page: TxPage) => Rx.of(RD.success(page))),
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/binance/service.ts:292]: im not sure whether we need a `mergeMap` here ? is not plain `map` enough ?

File: src/renderer/services/binance/service.ts [hunk 3]
```diff
@@ -296,16 +312,16 @@ const { stream$: reloadTxsSelectedAsset$, trigger: reloadTxssSelectedAsset } = t
  */
 const txsSelectedAsset$: Observable<TxsRD> = Rx.combineLatest(
   client$,
-  reloadTxsSelectedAsset$.pipe(debounceTime(300)),
+  loadSelectedAssetTxs$.pipe(debounceTime(300)),
   selectedAsset$
 ).pipe(
-  mergeMap(([client, _, oAsset]) => {
+  mergeMap(([client, { limit, offset }, oAsset]) => {
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/binance/service.ts:318]: seems we can use `switchMap` here ?

**Review by thatStrangeGuyThorchain — APPROVED** [review]

File: src/renderer/components/wallet/AssetDetails.tsx (hunk 1 above)
Comment (veado) [reply src/renderer/components/wallet/AssetDetails.tsx:146]: Good point. `MAX_PAGES` is misleading here, it should be `MAX_ROWS` or `MAX_ITEMS`. I'm going to change it ...

**Pull request merged**